chains/v4/chains.json

```diff
@@ -3015,6 +3015,101 @@
                 "symbol": "TUR",
                 "precision": 10,
                 "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Turing.svg"
+            },
+            {
+                "assetId": 1,
+                "symbol": "KSM",
+                "precision": 12,
+                "priceId": "kusama",
+                 "type": "orml",
+                "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Kusama.svg",
+                "typeExtras": {
+                    "currencyIdScale": "0x01",
+                    "currencyIdType": "turing_runtime.CurrencyId",
+                    "existentialDeposit": "10000000000",
```

Comment:
<img width="1169" alt="image" src="https://user-images.githubusercontent.com/570634/172867075-369b9921-1f44-4d93-873f-3d66192e05d9.png">

Existential deposits:

1) For KSM = 10^12 * 10^-5*10 = 10^8
2) For AUSD = 10^12 * 10^-2 =  10^10
3) For KAR = 10^12 * 10^-2*10 = 10^11
4) For LKSM = 10^12 * 10^-5*50 = 5 * 10^8
5) For HKO = 10^12 * 10^-2*50 = 5 * 10^11
6) For sKSM = 10^12 * 10^-5*50 = 5 * 10^8